### DISCUSSION TITLE
Weird Evil Behaviour

### DISCUSSION THREAD

**Discussion by harrycheon** [SystemCrafters/crafted-emacs#272] [Q&A] [answered]
Hi,

I've been using crafted-emacs for a few days now, enjoying learning and configuring emacs on my own.

There is one thing that is bugging me a bit. The evil keybinding `dj` is not acting like it does on vim.

Rather than deleting the current line and the line below it, `dj` deletes the contents from the current cursor until where the cursor would be if I were to press `j` and go down a line.

Is this intended and is there anyway to make it align the the behaviour in vim?

Thanks!

OS: macOS
Emacs Version: 28.2

P.S. I didn't make any changes to the evil configuration.

**Comment by harrycheon** [comment, marked as answer]
I found the part in the evil configuration that does this. Thanks.

**Reply by jeffbowman** [reply]
Do you mind sharing what you found please?  Just in case others come across this post and have the same question.
